src/client/clientmap.cpp

```diff
@@ -805,6 +805,27 @@ void MeshBufListMaps::addFromBlock(v3s16 block_pos, MapBlockMesh *block_mesh,
 	}
 }
 
+namespace {
+	// there is no convenient scope this would fit, so it's global
```

Comment:
A member of ClientMap? (Would still just be a single instance)
And actually maybe a CachedProfilerMetric (or something) might be a good construct to formalize for other scenarios.

Reply:
`transformBuffersToDrawOrder()` is not a member of ClientMap (would have to move a bunch of stuff to the header to facilitate that), so no good unfortunately.

> And actually maybe a CachedProfilerMetric (or something) might be a good construct to formalize for other scenarios.

I can see this "keep track of cache hits" pattern being used multiple times, but I don't think there are any other cases where the profiler is called "too often".